stock_production_lot_multi_company/__manifest__.py

```diff
@@ -1,13 +1,12 @@
 # -*- coding: utf-8 -*-
-#############################################################################
-# (c) 2015 Ainara Galdona - AvanzOSC
-# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
-#############################################################################
+# Copyright 2015 Ainara Galdona - AvanzOSC
+# Copyright 2017 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Stock Production Lot Multi Company",
     "summary": "Make serial numbers multi-company aware",
-    "version": "8.0.1.0.0",
+    "version": "10.0.0.1.0",
```

Comment:
10.0.1.0.0

Reply:
@pedrobaeza brief explanation: we are using setuptools-odoo to build "unstable" modules from open PRs. So we use versions < 1.0.0 for modules to be released. Now, assuming the module to be ready for merging, I change version to 1.0.0

Otherwise, we could directly use 1.0.0 for new PRs, then increase the version (1.0.1 ...) for fixes done during the review process.

What do you suggest?